.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature
+about: Define a new Gaffer feature
+title: ''
+labels: 'type-enhancement'
+assignees: ''
+
+---
+
+## What I'd like ##
```

Comment:
Let's try and make it simpler to fill in the user story "fields":

```
### Request ###
I am a Gaffer: 
<user> or <developer>

I would like:
<feature>
 
I would like it because: 
<reason>

### Specific feature conditions ###
For this feature to be satisfactory, it should work under these specific conditions:
- <condition/situation A>
- <condition/situation B>

### Technical Notes ###
Optionally, add any technical comments, notes, or requirements:
- <technical comment/note/requirement A>
- <technical comment/note/requirement B>
```